math/libqalculate/Portfile

```diff
@@ -3,8 +3,12 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        qalculate libqalculate 2.4.0 v
-revision            1
+github.setup        qalculate libqalculate 2.6.0a v
+
+# upstream played loose with release tag on 2.6
+version             2.6.0
+distname            ${github.project}-${version}b
+worksrcdir          ${github.project}-${version}
```

Comment:
Like I say in the comment, the upstream developer played loosy-goosy with the release tagging (https://github.com/Qalculate/libqalculate/releases). The description is simply 2.6. The release tag is 2.6.0a. The tarball is 2.6.0b. This is because the developer fixed a bug (specific to MacOS) and released a new tarball without making a new release (https://github.com/Qalculate/libqalculate/issues/96). It may not be pretty, but it works.

Reply:
Should we perhaps just consistently call it "b"?

Reply:
No. The 'a' is needed to get the correct release tag folder, the 'b' is needed to get the correct tarball, and setting the version to 2.6.0 is needed to get the correct worksrcdir. Also, upstream does not designate the version as 2.6.0b, only 2.6.0.

Reply:
Wow this is disgusting. Okay, I'm convinced you are doing the right thing.